From 76cbd296025731e1da2fe67963cf88385b0133b4 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Thu, 17 Sep 2020 18:00:38 +0200
Subject: [PATCH 1/1] mtd: fix typos in drivers/mtd/Kconfig

Fix a typo

%s/interract/interact/

Use Samsung's capitalization of their trademarks

%s/onenand/OneNAND/
%s/Hyperflash/HyperFlash/

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 drivers/mtd/Kconfig | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/mtd/Kconfig b/drivers/mtd/Kconfig
index 348b43e653..a77425d242 100644
--- a/drivers/mtd/Kconfig
+++ b/drivers/mtd/Kconfig
@@ -6,8 +6,8 @@ config MTD_PARTITIONS
 config MTD
 	bool "Enable MTD layer"
 	help
-	  Enable the MTD stack, necessary to interract with NAND, NOR,
-	  SPI-NOR, SPI-NAND, onenand, etc.
+	  Enable the MTD stack, necessary to interact with NAND, NOR,
+	  SPI-NOR, SPI-NAND, OneNAND, etc.
 
 config DM_MTD
 	bool "Enable Driver Model for MTD drivers"
@@ -98,7 +98,7 @@ config RENESAS_RPC_HF
 	bool "Renesas RCar Gen3 RPC Hyperflash driver"
 	depends on RCAR_GEN3 && DM_MTD
 	help
-	  This enables access to Hyperflash memory through the Renesas
+	  This enables access to HyperFlash memory through the Renesas
 	  RCar Gen3 RPC controller.
 
 config HBMC_AM654
-- 
2.28.0

